test/TestCase.h

```diff
@@ -102,7 +102,22 @@ class TestCase
 	bool m_shouldRun = true;
 };
 
-class EVMVersionRestrictedTestCase: public TestCase
+class CommonBaseTestCase: public TestCase
```

Comment:
Is there a reason you went with a new base class instead of implementing these additional functions in `TestCase`?

Reply:
At the beginning I thought it should be kind of mix-in class, but letter I've realised it's a base for all `TestCase` subclasses :)
@hrkrshnn Do you think worth to move the common implementation to the base `TestCase`?

Reply:
@ssi91 Every child of `TestClass` needs this, then I would prefer implementing these functions in `TestClass`.

Reply:
well, I've done this